### DISCUSSION TITLE
Wow~

### DISCUSSION THREAD

**Discussion by 01202008** [share121/inter-knot#4957] [灌水]
我中秋节放5天假！！！！！！

**Comment by sb35386057** [comment]
光顾着抽卡，忘了抽你了

**Comment by Liruijia666** [comment]
该打        我放8天😃

**Reply by DogOfFurina114514** [reply]
泥*************************

**Reply by cuijiaqi0406** [reply]
你们都不准玩地球online!!!!!!!

**Comment by DogOfFurina114514** [comment]
超！我放**3天**

**Comment by qingshuiglaze** [comment]
该死，没放假，甚至还在彩排军训汇演，那个好学校提前四天彩排啊喂

**Comment by wushang114** [comment]
巧了，我也是五天

**Comment by duoduo13123678** [comment]
666这个入开桂了（我只放三天）

**Comment by kinich-cute** [comment]
嗯？什么？

**Comment by Finkawa** [comment]
<img src="https://img-static.mihoyo.com/communityweb/upload/6b88559cbba96a1fa965e5412736370d.png" alt="兔躯一震" width="75" >**你该罚**（只放了三天的我)**QWQ**<img src="https://img-static.mihoyo.com/communityweb/upload/70b5b0c9543ee5080c6ae92fd2797892.png" alt="心情复杂" width="75" >

**Comment by yansai0v0** [comment]
我明天就要回去了😭😭😭😭😭😭

**Comment by shmily0818** [comment]
什么意思

**Comment by cuijiaqi0406** [comment]
你不准玩地球online!!!!!!!!!!!!!!

**Comment by wshtdgo** [comment]
你不许放假呜呜呜
